### PR TITLE
총수익/일간수익 토글 가능

### PR DESCRIPTION
**Pull request by seirion**

### CODE REVIEW

File: app/src/main/java/com/trueedu/project/model/dto/account/AccountResponse.kt [hunk 1]
```diff
@@ -138,4 +139,18 @@ data class AccountOutput2(
...
+    // 전날 대비 수익률
+    fun dailyProfitRate(): Double {
+        try {
+            val profit = assetChangeAmount.toDouble()
+            // FIXME: 예수금 포함이라서 정확하지 않음
+            val cost = previousTotalAssetEvaluationAmount.toDouble() // 전일 자산
+
+            if (cost == 0.0) return 0.0
+            return profit / cost * 100
+        } catch (_: Exception) {
```
Comment (falsetru) [inline app/src/main/java/com/trueedu/project/model/dto/account/AccountResponse.kt:152]: Exception 대신 더 구체적인 예외를 명시해주십쇼.

File: app/src/main/java/com/trueedu/project/ui/views/home/HomeViews.kt [hunk 2]
```diff
@@ -37,25 +38,56 @@ fun EmptyHome() {
...
+    }
+    val profitString = formatter.format(profit, true)
+
+    val profitRateString = if (dailyProfitMode) {
+        val dailyProfitRate = accountInfo.dailyProfitRate()
+        rateFormatter.format(dailyProfitRate, true)
+    } else {
+        val totalProfitRate = accountInfo.totalProfitRate()
+        rateFormatter.format(totalProfitRate, true)
+    }
```
Comment (falsetru) [inline app/src/main/java/com/trueedu/project/ui/views/home/HomeViews.kt:86]:
```suggestion
    val rate = if (dailyProfitMode) {
        accountInfo.dailyProfitRate()
    } else {
        accountInfo.totalProfitRate()
    }
    val profitRateString = rateFormatter.format(rate, true)
```

**Review by falsetru — APPROVED** [review]

File: app/src/main/java/com/trueedu/project/model/dto/account/AccountResponse.kt (hunk 1 above)
Comment (seirion) [reply app/src/main/java/com/trueedu/project/model/dto/account/AccountResponse.kt:152]: NumberFormatException 입니다 ㅈㅅ

Comment (falsetru) [reply app/src/main/java/com/trueedu/project/model/dto/account/AccountResponse.kt:152]: [`7759469` (#25)](https://github.com/seirion/true-project/pull/25/commits/775946910838555a155dcdeeaff79d1cf5a82f8c)

File: app/src/main/java/com/trueedu/project/ui/views/home/HomeViews.kt (hunk 2 above)
Comment (falsetru) [reply app/src/main/java/com/trueedu/project/ui/views/home/HomeViews.kt]: [`ebf6373` (#25)](https://github.com/seirion/true-project/pull/25/commits/ebf6373f165cde261088c6e004f285d3f19c427a)

**Review by falsetru — APPROVED** [review]

**Pull request merged**